Dockerfile

```diff
@@ -25,7 +25,8 @@ COPY --from=composer/composer:2-bin /composer /usr/local/bin/composer
 FROM base-dev as vendor
 COPY composer.json /fixer/composer.json
 WORKDIR /fixer
-RUN composer install --prefer-dist --no-dev --optimize-autoloader --no-scripts
+RUN composer remove --dev infection/infection --no-update \
```

Comment:
should we have a job on PR that verifies if docker image is buildable and runnable, similar like we do with phar file?

Reply:
In general, I personally would like to change our CI and instead of preparing PHP runtimes using `shivammathur/setup-php`, I would build Docker images and run all QA jobs on them (dev jobs on dev images, Fixer on release-like image). Then we would always know it's buildable and in sync with local development. But yeah, for now the job in PR that checks if Docker builds would be enough and really nice to see.